package.json

```diff
@@ -96,6 +96,11 @@
           "description": "Whether snapshot previews should show",
           "type": "boolean",
           "default": true
+        },
+        "jest.restartJestOnSnapshotUpdate": {
+          "description": "Restart Jest runner after updating the snapshots",
+          "type": "boolean",
+          "default": false
```

Comment:
I wanted this change to be not invasive so that current users do not get something they do not expect. I am open to making it a default if there is agreement.